### ISSUE TITLE
Training ends at epoch 660. 

### ISSUE THREAD

**Issue by yiping-tks** [shawLyu/Occ-SDF-Hybrid#2]
Hi, thanks for open sourcing this amazing work.
I have a question that my training process ends at epoch 660 (when I continue training, the result is the same) and the output infomation is "loss = 0.07628080993890762, rgb_loss = 0.016237832605838776, eikonal_loss = 0.015570213086903095, psnr = 31.56894874572754, bete=0.0011694224085658789, alpha=855.1230014707431", but the default end epoch is 2000, and the rendering is not so perfect, so how can I make the result better?
Thanks a lot! @shawLyu

**Comment by shawLyu** [comment]
Thanks for your interests in our work. Can you give me more details about the training dataset and representations like grids or mlp?

**Comment by yiping-tks** [comment]
Copy that!

**Comment by yiping-tks** [comment]
Thanks for your reply sincerely!
I have trained the model on scannet4 with 3090 GPU which has  24GB. I kept the network and other configurations the same as the scannet_mlp_hybrid.conf. The training process was normal before epoch 660, after that the terminal suddenly terminated. I tried to add the parameter "--is_continue", and it did start from epoch 660, but when it finished this epoch, problem poped up again.

**Comment by shawLyu** [comment]
I'm sorry for not getting back to you sooner. 
1. The 'self.max_total_iters' determines the total training steps, whose default value is 200K. To derive the training epochs, we calculate this based on the total training steps and the number of dataset images, which, in the case of Scannet 4, amounts to 660. The newly loaded checkpoints will inherit the training step, achieving 200K after training. To prolong your training, you can change the parameters mentioned above and continue the training process.
2. Unfortunately, for the neural-based reconstruction methods, we have to sacrifice the quality of novel view synthesis to get better-reconstructed results. You may find the rendering qualities are blurred or over smoothness which may be caused by the surface smoothness constraint.
